src/components/pages/TodoPage.js

```diff
@@ -24,9 +24,28 @@ export default class TodoPage extends Component {
                 Completed
               </th>
             </tr>
+            <tr>
+              <th>
+
```

Comment:
add some mock data to this first row so it looks like a real todo, ie.e give it a title and description, don't worry about the completed field right now